deploy/Vagrantfile

```diff
@@ -583,6 +582,104 @@ Vagrant.configure(2) do |config|
     server.vm.network :private_network, ip: "10.0.1.29"
   end
 
+  # DB Server
+  config.vm.define :db_server_centos7 do |server|
+    server.vm.box = "centos/7"
+    server.vm.host_name = "db7.test.dev"
+
+    server.ssh.forward_agent = true
+
+    server.vm.provider "virtualbox" do |v|
+      v.memory = 1024
+      v.cpus = 1
+    end
+
+    server.ssh.insert_key = false
+    server.vm.provision "shell", inline: <<-SHELL
+      # bug with vagrant 1.9+ and centos/7
+      sudo ifup eth1
+    SHELL
+
+    server.vm.provision "ansible" do |ansible|
+
+      if ENV['DB_SERVER_EXTRA_VARS']
+        ansible_extra_vars = ENV['DB_SERVER_EXTRA_VARS']
+      else
+        print "To use the db_server VM, set the DB_SERVER_EXTRA_VARS environment variable to the name of a local YAML file that contains pp_web_username (email addresses only) and pp_web_password.\n"
+        print "Example:\n"
+        print "$ export DB_SERVER_EXTRA_VARS=local_settings.yml\n"
+        print "$ cat local_settings.yml\n"
+        print "---\n"
+        print "pp_web_username: myusername@example.com\n"
+        print "pp_web_password: mypassword\n"
+      end
```

Comment:
This section is unnecessary when installing the open source version, since `pp_web_username` and `pp_web_password` are only needed for EDB.